Core/NoOpRunner.Core/Shapes/PowerUp.cs

```diff
@@ -31,5 +30,15 @@ private Color GetPowerUpColor(PowerUps powerup)
                 default: return Color.Black;
             }
         }
+
+        public override bool CanOverlap(BaseShape other)
+        {
+            return false; // Not Implemented Yet
```

Comment:
You can already implement this part if you want to, just return true if other is Player or Rocket, False otherwise (will need to be updated when we add more Entities)

Reply:
Fixed